src/plugins/ec.ts

```diff
@@ -172,6 +173,32 @@ export class ECPlugin extends BasePlugin {
             }, {});
     }
 
+    private convertProductPriceToNumber(product: Product) {
+        if (product.price) {
+            product.price = this.tryConvertStringPriceToNumber(product.price);
+        }
+
+        return product;
+    }
+
+    private convertImpressionPriceToNumber(impression: BaseImpression) {
+        if (impression.price) {
+            impression.price = this.tryConvertStringPriceToNumber(impression.price);
+        }
+
+        return impression;
+    }
+
+    private tryConvertStringPriceToNumber(price: number | string): number | string {
+        if (typeof price === 'number') {
+            return price;
+        }
+
+        let parsedPrice = parseFloat(price.replace(/[^0-9\.-]/g, ''));
```

Comment:
This can be a `const` since it isn't modified after its declaration
```suggestion
        const parsedPrice = parseFloat(price.replace(/[^0-9\.-]/g, ''));
```